e2e_test.ts

```diff
@@ -34,55 +34,119 @@ import options from "./fresh.config.ts";
  */
 const handler = await createHandler(manifest, options);
 
-function assertResponseNotFound(resp: Response) {
+function assertResponseAccepted(resp: Response) {
+  assert(resp.ok);
+  assertEquals(resp.body, null);
+  assertEquals(resp.status, Status.Accepted);
+}
```

Comment:
I don't think this, and other assertions that solely check response status, are needed. Instead, one can just do `assertEquals(resp.status, status)`. Custom assertions are not needed.

Also, checking the body and `ok` are not needed. I'd go as far as to say that if `resp.status` is checked, `resp.ok` never needs to be checked.

Reply:
midway through the first assertions i got carried away.
having the defined ruleset helped a lot!